EventHubs/src/azuredeploy_metrics.json

```diff
@@ -3,30 +3,30 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "sites_SumoMetricsFunctionApp_name": {
-            "defaultValue": "[concat('SumoMetricsFuncApp', uniqueString(resourceGroup().id))]",
+            "defaultValue": "[concat('SMFuncApp', uniqueString(resourceGroup().id))]",
             "type": "String"
         },
-        "namespaces_SumoMetNamespace_name": {
-            "defaultValue": "[concat('SumoMetricsNamespace', uniqueString(resourceGroup().id))]",
+        "namespaces_SumoMetricsNamespace_name": {
```

Comment:
why do we need to change parameter name, is there some test failing ? we may have to update the documentation as well

Reply:
updated